packages/ui/DeleteDialog/index.tsx

```diff
@@ -43,7 +46,7 @@ export function DeleteDialog({
           }}>
           {useSaveInsteadOfCancel ? "Save" : "Cancel"}
         </Button>
-        <Button variant="warn" onClick={onDelete} loading={isDeleting} disabled={disabled}>
```

Comment:
why did you remove the disabled flag? Since this component is use in several locations removing it might break functionality somewhere else. Did you run a build locally to confirm that it doesnt?

Reply:
The disabled prop value was undefined